documentation/book/ref-kafka-authorization.adoc

```diff
@@ -25,4 +24,20 @@ To enable simple authorization, the `type` field should be set to `simple`.
 authorization:
   type: simple
 # ...
-----
\ No newline at end of file
+----
+
+== Super users
+
+Super users can access all resources in your Kafka cluster regardless of any access restrictions defined in ACLs.
+To designate super users for a Kafka cluster, enter a list of user principles, delimited by semi-colons, in the `superUsers` field.
```

Comment:
I think we should add the information that the users using TLS authentication are prefixed with the `CN=` prefix.